CHANGELOG.md

```diff
@@ -129,6 +129,9 @@ Magento Functional Testing Framework Changelog
 2.6.3
 -----
 
+### Fixes
```

Comment:
@okolesnyk isn't the added content from 2.6.4? I believe we don't have fixes + documentation from 2.6.4 onwards in 3.0.0-GA

Reply:
So do I need to remove this change?

Reply:
will fix it on 3.0.0-GA, 2.6.3 and 2.6.4 changelogs were clubbed